pkg/kubelet/cm/topologymanager/policy.go

```diff
@@ -30,6 +30,14 @@ type Policy interface {
 	Merge(providersHints []map[string][]TopologyHint) (TopologyHint, bool)
 }
 
+// IsAlignmentGuaranteed return true if the given policy guarantees that either
+// the compute resources will be allocated within a NUMA boundary, or the allocation will fail at all.
+func IsAlignmentGuaranteed(p Policy) bool {
+	// TODO: should we add another field to the Policy interface to explicitly convey the alignment promise?
```

Comment:
I have hard time judging if this TODO is fine to keep. If we do not plan to fix it any time soon, perhaps it can be rephrased into the explanation of the edge cases?

Reply:
good point, rephrasing